apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -161,14 +161,22 @@ function wpcom_global_styles_enqueue_block_editor_assets() {
 		true
 	);
 	wp_set_script_translations( 'wpcom-global-styles-editor', 'full-site-editing' );
-	$plan = wpcom_site_has_global_styles_in_personal_plan() ? 'personal-bundle' : 'value_bundle';
+	$is_global_styles_in_personal_plan = wpcom_site_has_global_styles_in_personal_plan();
+	$plan                              = $is_global_styles_in_personal_plan ? 'personal-bundle' : 'value_bundle';
+
+	$reset_global_styles_support_url = 'https://wordpress.com/support/using-styles/#reset-all-styles';
+	if ( class_exists( 'WPCom_Languages' ) ) {
```

Comment:
Should we define a default value for cases where `WPCom_Languages` does not exist?

Reply:
The default value is implicit, it's the original URL (English support documentation), we only localize it if `WPCom_Languages` is available.

Reply:
oh! I think I didn't note we were initializing the var with "https://wordpress.com/support/using-styles/#reset-all-styles". I somehow missed that I was thinking if was unset.